code/game/gamemodes/vampire/vampire_powers.dm

```diff
@@ -22,11 +22,11 @@
 		to_chat(user, "<span class='warning'>You are in too weak of a form to do this!</span>")
 		return 0
 
-	var/datum/vampire/vampire = user.mind.vampire
-
-	if(!vampire)
+	if(!is_vampire(user))
 		return 0
 
+	var/datum/antagonist/vampire/vampire = user.mind.has_antag_datum(/datum/antagonist/vampire)
```

Comment:
Since you're accessing the contents of the `vampire` datum here, it'd be better do move this above the `if(!vampire)` check instead of placing an extra `is_vampire` check.